public/xmlns/tract.xsd

```diff
@@ -219,7 +219,8 @@
                         </xs:documentation>
                     </xs:annotation>
                 </xs:attribute>
-                <!-- control-color: DEFAULT "GodTools blue" rgba(59,164,219,1) . -->
+                <!-- control-color: iOS DEFAULT is "GodTools blue" rgba(59,164,219,1) , Android DEFAULT is the primary-color attribute set at the
+                nearest ancestor either page, or manifest. -->
```

Comment:
We will need to standardize this for the spec. I would argue it should default to a theme color defined at a higher level. Also, the default color comment should be included within the `xs: documentation` node

Reply:
i'm looking at standardizing it to primary-color now.

Reply:
good deal :)

Reply:
Fixed comment and updated iOS renderer here: 
https://github.com/CruGlobal/godtools-swift/commit/e4f76d49511783d64e9d4d5ebb00393e2cf06fbe